supervision/tracker/byte_tracker/core.py

```diff
@@ -249,12 +252,16 @@ def update_with_detections(self, detections: Detections) -> Detections:
             detections.confidence = np.array(
                 [t.score for t in tracks], dtype=np.float32
             )
+            detections.mask = np.array([t.mask for t in tracks], dtype=bool)
+
         else:
             detections.tracker_id = np.array([], dtype=int)
 
         return detections
 
-    def update_with_tensors(self, tensors: np.ndarray) -> List[STrack]:
+    def update_with_tensors(
+        self, tensors: np.ndarray, masks: np.ndarray = None
+    ) -> List[STrack]:
         """
         Updates the tracker with the provided tensors and returns the updated tracks.
```

Comment:
Please update the docstring to reflect the API change. Add and describe `masks` argument.